### PR TITLE
Add support for sending comments tagged with the users to be notified.

### PR DESCRIPTION
**Pull request by seema-at-box**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @seema-at-box has signed the CLA. Thanks for the pull request!

File: box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestCommentAdd.java [hunk 1]
```diff
@@ -47,6 +47,11 @@ public R setMessage(String message) {
         return (R) this;
     }
```
Comment (brianytang) [inline box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestCommentAdd.java]: should have javadocs for all public methods

**Comment by brianytang** [comment]
+1

**Comment by doncung** [comment]
+1

**Pull request merged**